### PR TITLE
TILA-1917 | Expose date of birth to gql with access logging

### PR DESCRIPTION
**Pull request by nicobav**
Adds PersonalInfoViewLog model to store views of a users date_of_birth.
 - own views does not get logged

Adds date_of_birth to UserType and adds new gql query for querying specific user with UserPermissions (so even regular user cannot use this endpoint to view own info and should be using currenUser)
 - logs view to personal view log when this field is requested
Adds reservee_date_of_birth to reservationType  - with reservation read permissions :warning: 
 - logs view to personal view log when this field is requested.

### CODE REVIEW

**Comment by nicobav** [comment]
seems I screwed a rebase with one of the payment commits => removing that one.

**Review by saaste — APPROVED** [review]
This turned out to be extremely nice! Noooice! 👏

File: api/graphql/tests/base.py [hunk 1]
```diff
@@ -24,3 +34,74 @@ def setUpTestData(cls):
...
+            role=ServiceSectorRoleChoice.objects.get(code="admin"),
+            permission="can_handle_applications",
+        )
+
+        return service_sector_admin
+
+    def create_unit_admin(self, unit=None):
+        unit_admin = get_user_model().objects.create(
+            username="unit_admin",
+            first_name="Amin",
```
[the remark this answers is not in the document]
Comment (nicobav) [reply api/graphql/tests/base.py:64]: yup, agreed, will change those :+1:

**Comment by nicobav** [comment]
@saaste I removed the permission check for the curren_user => Its somewhat unnecessary  since we're fetching only the user who's making the request.

**Pull request merged**